torch/csrc/autograd/custom_function.cpp

```diff
@@ -3,6 +3,21 @@
 
 namespace torch { namespace autograd {
 
+VariableInfo::VariableInfo(const Variable& var)
```

Comment:
No substantive change here, is that right? Just code motion?

Reply:
yes